contrib/config/apollo/watcher.go

```diff
@@ -14,26 +18,40 @@ type customChangeListener struct {
 	event chan []*config.KeyValue
```

Comment:
How about using in&out channel to distinguish these `watcher` and `listener` channels?

Reply:
done